pymodaq_plugins/hardware/keithley2110/readme.linux.md

```diff
@@ -0,0 +1,8 @@
+#keithley instruments
+
+## udevrules
+User have to be in the group "dialout"
```

Comment:
What do you mean by these rules?

Reply:
as explained in the [pyvisa py doc](https://pyvisa-py.readthedocs.io/en/latest/installation.html#how-do-i-know-if-pyvisa-py-is-properly-installed), under linux "modify udev rules to allow non-root access to the device you are trying to connect to".

Reply:
maybe it's better if all visa devices go in a group called visa ?

Reply:
ok, I'm not fluent enough in Linux to advise on this... I'll trust you :-)